content/en/debugging.md

```diff
@@ -0,0 +1,14 @@
+---
+title: Debugging
+status: Feedback Appreciated
```

Comment:
I changed it to feedback appreciated as it still needs to be reviewed by another approver